tools/configure_server_settings.rb

```diff
@@ -4,19 +4,26 @@
 
 opts = Trollop.options(ARGV) do
   banner "USAGE:   #{__FILE__} -s <server id> -p <settings path separated by a /> -v <new value>\n" \
-         "Example: #{__FILE__} -d -s 1 -p reporting/history/keep_reports -v 42"
```

Comment:
hmm, was this wrong previously?  Perhaps this example should also specify a -i

Reply:
added new line because a 3rd example was added below with `-i`